packages/near-fast-auth-signer-e2e-tests/scripts/faucet.js

```diff
@@ -0,0 +1,87 @@
+const { CovalentClient } = require('@covalenthq/client-sdk');
+const { ethers } = require('ethers');
+
+// These are the derived addresses through the tests that needs to receive some tokens from a faucet periodically, same thing and the commented ones below
+const ETH_PERSONAL_KEY_ADDRESS = '0xf64750f13f75fb9e2f4d9fd98ab72d742d1e33eb';
+const BNB_DOMAIN_KEY_ADDRESS =  '0x81d205120a9f04d3f1ce733c5ed0a0bc66714c71';
+
+function fromWei(wei) {
+  return ethers.formatEther(wei.toString());
+}
+async function fetchEVMWalletBalance(
+  address,
+  chain
+) {
+  const client = new CovalentClient(process.env.COVALENT_API_KEY);
+  const response = await client.BalanceService.getTokenBalancesForWalletAddress(
+    chain,
+    address
+  );
+
+  if (response.error) {
+    throw new Error(`Failed to fetch balance: ${response.error_message}`);
+  }
+
+  const { balance } = response.data.items[0];
+  if (balance === null) return 0;
+  return fromWei(BigInt(balance).toString());
+}
+
+function getWalletChain(chain) {
+  switch (chain) {
+    case 'eth':
+      return 'sepolia';
+    case 'bnb':
+      return 'bnb-testnet';
+    default:
+      return null;
+  }
+}
+
+async function fillWallet(address, chain) {
+  const walletChain = getWalletChain(chain);
+
+  if (!walletChain) {
+    throw new Error(`Unsupported chain: ${chain}`);
+  }
+
+  const apiUrl = `https://api.chainstack.com/v1/faucet/${walletChain}`;
+
+  const response = await fetch(apiUrl, {
+    method:  'POST',
+    headers: {
+      Authorization:  `Bearer ${process.env.CHAINSTACK_API_KEY}`,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ address }),
+  });
+
+  if (!response.ok) {
+    return response.text().then((text) => { throw new Error(text); });
+  }
+
+  console.log(`Testnet token request for ${chain} address ${address} successful`);
+
+  return response.json();
+}
+
+async function requestTokensFromFaucet() {
```

Comment:
Shouldn't we make the `address/chain` arguments of the function instead of hard code taht we will fetch only for ETH/BNB? 

So, in the future, will be easier to add test for more chains

Reply:
Done